docs/development_checklist.md

```diff
@@ -8,7 +8,7 @@ Before you commit any changes, check your work against these requirements:
 - All communication to external devices or services must be wrapped in an external Python library hosted on [pypi](https://pypi.python.org/pypi).
 - New dependencies are added to `requirements_all.txt` (if applicable), using `python3 -m script.gen_requirements_all`
 - New codeowners are added to `CODEOWNERS` (if applicable), using `python3 -m script.hassfest`
-- The `.coveragerc` file is updated to exclude your platform if there are no tests available or your new code uses a third-party library for communication with the device, service, or sensor
+- The `.coveragerc` file is updated to include at least your platforms `config_flow.py` and has 100% coverage for this file. You can exclude your platform if there are no tests available or your new code uses a third-party library for communication with the device, service, or sensor. 
```

Comment:
This is confusing now:

"The `.coveragerc` file is updated to include at least your platforms `config_flow.py`"

It is very easy to misread that it needs to be included (as in listed) in that file; which of course it should not.

Reply:
Thanks, adjusted to just add `config_flow.py can't be excluded as it must be fully tested (100% coverage).`